radiodns/apps/stations/models.py

```diff
@@ -18,14 +18,14 @@ class Station(models.Model):
         LogoImage, on_delete=models.SET_NULL, blank=True, null=True
     )
 
-    radiovis_enabled = models.BooleanField(default=False)
-    radiovis_service = models.CharField(max_length=255, blank=True)
-    radioepg_enabled = models.BooleanField(default=False, blank=True)
-    radioepgpi_enabled = models.BooleanField(default=False)
-    radioepg_service = models.CharField(max_length=255, blank=True)
-    radiotag_enabled = models.BooleanField(default=False)
-    radiotag_service = models.CharField(max_length=255, blank=True)
-    radiospi_enabled = models.BooleanField(default=False)
+    radiovis_enabled = models.BooleanField(default=False, null=True)
+    radiovis_service = models.CharField(max_length=255, blank=True, null=True)
+    radioepg_enabled = models.BooleanField(default=False, blank=True, null=True)
+    radioepgpi_enabled = models.BooleanField(default=False, null=True)
+    radioepg_service = models.CharField(max_length=255, blank=True, null=True)
+    radiotag_enabled = models.BooleanField(default=False, null=True)
+    radiotag_service = models.CharField(max_length=255, blank=True, null=True)
+    radiospi_enabled = models.BooleanField(default=False, null=True)
```

Comment:
I am a bit worried about having null values for those boolean fields. It would be good to check whether it doesn't break any other functionalities.

Reply:
Okay, I can correct this by changing the incriminated lines in the dump.